requirements_test.txt

```diff
@@ -22,5 +22,4 @@ setuptools-scm==5.0.0 ; python_version == '2.7'
 wheel==0.36.1 ; python_version == '2.7'
 moto == 0.4.31
 responses >= 0.5.1, < 0.10.3
-requests >= 2.9.0
```

Comment:
We already vend requests, and attempting to use pip to install requests on some distros/versions of pythons fails because it is part of dist-utils. This causes the pip install -r requirements_test.txt to bail out.